src/app/signalhandler.cpp

```diff
@@ -36,11 +36,9 @@
 #include <csignal>
 #include <tuple>
 
-#ifndef Q_OS_WIN
-#ifndef Q_OS_HAIKU
```

Comment:
I don't know what `Q_OS_HAIKU` would expect here. So I just ignore it and go for the clean code.
qbt on haiku is really outdated: https://depot.haiku-os.org/#!/pkg/qbittorrent?bcguid=bc648-ICWG
So I suppose they would need to supply some patches to us if they want to get it working anyway.